jenkinsfile-runner-tests/tests.sh

```diff
@@ -12,11 +16,69 @@ oneTimeSetUp() {
 
   echo "Moving CLI jar file to $current_directory"
   find $test_framework_directory/source/custom-war-packager-cli/target -name '*-with-dependencies.jar' -exec mv {} $current_directory/cwp.jar ';'
+  cwp_jar="$current_directory/cwp.jar"
+}
+
+#
+# Use a JCasC configuration to build the JFR image, then get the system message that was set in
+# casc.yml to verify that it was applied.
+#
+# TODO for this and other tests we are copying casc.yml to the current directory so that it is
+# referenced properly in packager-config-yml.  It would be better to put an environment variable
+# or placeholder in packager-config.yml and update that with the real location before running.
+#
+test_cwp_casc_simple() {
+  cp $current_directory/test_resources/test_cwp_casc_simple/casc.yml .
+  jfr_tag=$(execute_cwp_jar_and_generate_docker_image "$working_directory" "${cwp_jar}" "$version" "$current_directory/test_resources/test_cwp_casc_simple/packager-config.yml" "$jenkinsfile_runner_tag" | grep 'Successfully tagged')
+  execution_should_success "$?" "$jenkinsfile_runner_tag" "$jfr_tag"
+
+  result=$(run_jfr_docker_image_with_jfr_options "$jenkinsfile_runner_tag" "$current_directory/test_resources/test_cwp_casc_simple/Jenkinsfile" "--no-sandbox")
+  jenkinsfile_execution_should_succeed "$?" "$result"
+  assertContains "Should contain the system message configured by CasC" "$result" "Jenkins configured automatically by Jenkins Configuration as Code Plugin"
```

Comment:
See https://github.com/jenkinsci/jenkinsfile-runner-test-framework/blob/4ed3c35650fe880345b77ee4776b29bd00944fb3/src/hooks/logs.inc#L7-L37, that can be used in all your tests
```suggestion
  logs_contains "Jenkins configured automatically by Jenkins Configuration as Code Plugin" "$result"
```

Reply:
Or even without storing and using $result later.